manim/scene/scene.py

```diff
@@ -1030,12 +1030,28 @@ def get_run_time(self, animations: list[Animation]):
         float
             The total ``run_time`` of all of the animations in the list.
         """
+        max_run_time = 0
+        frame_rate = (
+            1 / config.frame_rate
+        )  # config.frame_rate holds the number of frames per second
+        for animation in animations:
+            if animation.run_time <= 0:
+                raise ValueError(
+                    f"{animation} has a run_time of <= 0 seconds which Manim cannot render. "
+                    "Please set the run_time to be positive."
+                )
+            elif animation.run_time < frame_rate:
+                logger.warning(
+                    f"Original run time of {animation} is shorter than current frame "
+                    f"rate (1 frame every {frame_rate:.2f} sec.) which cannot be rendered. "
+                    "Rendering with the shortest possible duration instead."
+                )
```

Comment:
With the current way this is implemented, doing something like
```py
self.play(
    Succession(
        Create(Circle()),
        Create(Star()),
        lag_ratio=1,
    ),
    run_time=1e-3
)
```
Results in two warnings. It would be nice if this could be fixed, maybe in a follow-up PR?

Reply:
Interesting, not sure I can explain this intuitively. Would need to investigate; sounds like a job for a follow-up PR. 🧐